File: src/main/java/br/com/leonardoferreira/jirareport/repository/LeadTimeRepository.java
package br.com.leonardoferreira.jirareport.repository;

import br.com.leonardoferreira.jirareport.domain.LeadTime;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface LeadTimeRepository extends CrudRepository<LeadTime, Long> {

    void deleteByIssueId(Long id);

}

File: database-library/src/main/kotlin/br/com/jiratorio/repository/LeadTimeRepository.kt
package br.com.jiratorio.repository

import br.com.jiratorio.domain.entity.LeadTimeEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface LeadTimeRepository : CrudRepository<LeadTimeEntity, Long>, LeadTimeNativeRepository {

    fun deleteByIssueId(id: Long)

}
